src/test/java/org/codehaus/mojo/license/LicenseMojoUtilsTest.java

```diff
@@ -0,0 +1,162 @@
+package org.codehaus.mojo.license;
+
+import static org.junit.Assert.*;
+
+import java.io.File;
+
+import org.junit.Test;
+
+public class LicenseMojoUtilsTest
+{
+    private String resolvedUrl;
+    private File deprecatedFile;
+    private String url;
+    private File basedir = new File( "" );
+    private MockLogger log = new MockLogger();
+
+    @Test
+    public void testIsValidNull()
+    {
+        assertFalse( LicenseMojoUtils.isValid( null ) );
+    }
+
+    @Test
+    public void testIsValidEmpty()
+    {
+        // TODO is this correct?
```

Comment:
Yes, let's assume it is correct unless proven otherwise. Please remove the TODO.

Reply:
I've added a comment: `This might be wrong; feel free to change the test when it starts to fail` so the next person will not hesitate to fix the test.